test/unit/pjax_fallback.js

```diff
@@ -313,6 +313,50 @@ asyncTest("POST with data object"+s, function() {
   })
 })
 
+asyncTest("GET with data array"+s, function() {
+  var frame = this.frame
+
+  this.loaded = function() {
+    equal(frame.location.pathname, "/env.html")
+    equal(frame.location.search, "?foo=bar")
+
+    var env = JSON.parse(frame.$("#env").text())
+    equal(env['REQUEST_METHOD'], "GET")
+    equal(env['rack.request.query_hash']['foo'], 'bar')
```

Comment:
Could you expand these tests so that at least 2 values get submitted, and test the entire values of `query_hash/form_hash` objects with `deepEqual()`? Thanks